app/templates/orders/order_block.hbs

```diff
@@ -19,64 +19,66 @@
 
     </div>
 
-    <div class="row order_body">
+    {{#unless smallOrderBlocks}}
```

Comment:
The github diff looks a bit messy, but {{#unless smallOrderBlocks}} is the only thing I've added